postgres/10.0-alpine/README.md

```diff
@@ -0,0 +1,36 @@
+The [Postgres](https://www.postgresql.org) relational database management system has a feature called [logical decoding](https://www.postgresql.org/docs/9.4/static/logicaldecoding-explanation.html) that allows clients to extract all persistent changes to a database's tables into a coherent, easy to understand format which can be interpreted without detailed knowledge of the database's internal state. An output plugin transform the data from the write-ahead log's internal representation into the format the consumer of a replication slot desires.
+
+This image is based upon [`postgres:10.0-alpine`](https://hub.docker.com/_/postgres/) and adds a Debezium-specific [logical decoding output plugin that uses Protobuf](https://github.com/debezium/postgres-decoderbufs), enabling the [Debezium PostgreSQL Connector](http://debezium.io/docs/connectors/postgresql/) to capture changes committed to the database and record the data change events in Kafka topics. The image also includes [PostGIS](http://www.postgis.net) spatial database extender used to provide geospatial queries, so that changes to geometric data can also be captured by Debezium.
```

Comment:
I suppose it's copied from the existing one, but it'd be good to clarify that there are two LD plug-ins actually by now: the ProtoBuf-based one and another one based on wal2json.